### PR TITLE
[ci] test azp coverage artifact logging 

### PR DESCRIPTION
**Pull request by asraa**
Signed-off-by: Asra Ali <asraa@google.com>

Commit Message: Adds a log error message when coverage fails that points to the coverage report produced.
image

Risk Level: Low 
Testing: Manually testing

### CODE REVIEW

**Comment by asraa** [comment]
still wip, will ping when ready

**Comment by asraa** [comment]
Ready for review

File: test/run_envoy_bazel_coverage.sh [hunk 1]
```diff
@@ -93,6 +93,7 @@ if [[ "$VALIDATE_COVERAGE" == "true" ]]; then
   fi
   COVERAGE_FAILED=$(echo "${COVERAGE_VALUE}<${COVERAGE_THRESHOLD}" | bc)
   if [[ "${COVERAGE_FAILED}" -eq 1 ]]; then
+      echo "##vso[task.setvariable variable=COVERAGE_FAILED]$COVERAGE_FAILED"
```
Comment (lizan) [inline test/run_envoy_bazel_coverage.sh:96]:
```suggestion
      echo "##vso[task.setvariable variable=COVERAGE_FAILED]${COVERAGE_FAILED}"
```

per bash style guide, ditto below.

Comment (asraa) [reply test/run_envoy_bazel_coverage.sh]: done, thank you

**Comment by asraa** [comment]
friendly ping @lizan ?

**Review by lizan — APPROVED** [review]

**Pull request merged**